scripts/asm-installer/install_asm

```diff
@@ -953,6 +969,10 @@ validate_args() {
     if [[ "${USE_VM}" -eq 1 ]]; then
       fatal "Adding VM workloads with managed control plane is not supported."
     fi
+
+    if [[ "${CUSTOM_REVISION}" -eq 1 ]]; then
+      fatal "Adding custom revision label with managed control plane is not supported"
```

Comment:
I think "Specifying a revision label" is a bit closer to what's happening, instead of "Adding custom revision label".